examples/lib/stories/camera_and_viewport/zoom.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/gestures.dart';
+
+class ZoomGame extends BaseGame with ScrollDetector, ScaleDetector {
+  final Vector2 viewportResolution;
+  late SpriteComponent flame;
+
+  Vector2? lastDelta;
+
+  ZoomGame({
+    required this.viewportResolution,
+  });
+
+  @override
+  Future<void> onLoad() async {
+    final flameSprite = await loadSprite('flame.png');
+
+    viewport = FixedResolutionViewport(viewportResolution);
+
+    final flameSize = Vector2(149, 211);
+    add(
+      flame = SpriteComponent(
+        sprite: flameSprite,
+        size: flameSize,
+      )..anchor = Anchor.center,
+    );
+    camera.followComponent(
+      flame,
+      relativeOffset: Anchor.center.toVector2(),
+    );
+  }
+
+  static const zoomPerScrollUnit = 0.001;
+  @override
+  void onScroll(PointerScrollInfo event) {
+    camera.zoom += event.scrollDelta.game.y * zoomPerScrollUnit;
```

Comment:
you should multiple/divide the zoom by the delta abs right?

Reply:
Not sure I understood that, could you give me a small snippet example?

Reply:
I am updating the code of isomash to use this and I can share that (working through another camera issue I found there rn, maybe we have a bug somewhere)

Reply:
but what I meant was to multiply instead of add

zoom = event.scrollDelta.game.y * zoomPerScrollUnit
camera.zoom = if (zoom.sign < 1) camera.zoom / zoom.abs else camera.zoom * zoom.abs

so the effect scales with the current scales (accelerates in the global reference frame)

Reply:
Hmm, not really sure I understood the rational, but I will test and I may understand better when coding.

Reply:
I tried that idea but the scroll zoom got even weirder, on the first zoom, the sprite simply disappear :(